apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorContents.vue

```diff
@@ -217,8 +223,19 @@ const handleChangeToolbox = (options) => {
 const handleClickListItem = (detailLink) => {
     SpaceRouter.router.push(detailLink);
 };
-const handleRefreshCollectorList = () => {
-    emit('refresh-collector-list');
+
+/* API */
+const handleClickCollectData = async (collectorId) => {
+    state.collectLoading = true;
+    try {
+        await collectorPageStore.restartCollectJob(collectorId);
+        refreshCollectorList();
+        showSuccessMessage(i18n.t('INVENTORY.COLLECTOR.CREATE.ALT_S_COLLECT_EXECUTION'), '');
+    } catch (e) {
+        ErrorHandler.handleError(e);
```

Comment:
issue: In this case, must call `handleRequestError` to popup toast.

Reply:
issue: Actually, this catch block will never be called because `restartCollectJob` action catches an error.
To fix this issue, do not catch an error at `restartCollectJob`.